cli/cmd/runner.go

```diff
@@ -31,6 +31,7 @@ type runners struct {
 	dir              string
 	outputFormat     string
 	w                *tabwriter.Writer
+	werr             *tabwriter.Writer
```

Comment:
What is the use for this?

Reply:
Related to your other comment. If we don't send the supported distro's on validation error to stderr, we will not need this one.

Reply:
Removed